src/sidebar/panel.js

```diff
@@ -6,12 +6,17 @@ const formats = [
   'strike',
   'header',
   'indent',
-  'list'
+  'list',
+  'direction',
+  'align'
 ];
+const UI_LANG = browser.i18n.getUILanguage();
+const RTL_LANGS = ['ar', 'fa', 'he'];
+const LANG_DIR = RTL_LANGS.indexOf(UI_LANG) === -1 ? 'ltr' : 'rtl';
```

Comment:
Nit: you can use the ES6-ish [`Array#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) instead of checking against -1 index.